recipes/cerberecontrib_ghrsst/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set python_min = "3.10" %}
+{% set name = "cerberecontrib-ghrsst" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerberecontrib-ghrsst/-/archive/{{ version }}/cerberecontrib-ghrsst-{{ version }}.tar.gz
+  sha256: 7f84aeb91bb181c20b2ebe9f9405407d0f4717a187e66f4366eb4ad49cd970c3
+
+build:
+  noarch: python
+  script_env:
+    - POETRY_DYNAMIC_VERSIONING_BYPASS={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning >=1.0.0,<2.0.0
+    - pip
+  run:
+    - python {{ python_min }}
```

Is this py310 only? If not you need the `>=`.

```suggestion
    - python >={{ python_min }}
```

---

Yes it is py310 only.